handlers/twiml/twiml.go

```diff
@@ -43,6 +43,14 @@ var (
 	twilioBaseURL = "https://api.twilio.com"
 )
 
+// see https://www.twilio.com/docs/sms/accepted-mime-types#accepted-mime-types
+var mediaSupport = map[handlers.MediaType]handlers.MediaTypeSupport{
+	handlers.MediaTypeImage:       {MaxBytes: 5 * 1024 * 1024},
+	handlers.MediaTypeAudio:       {MaxBytes: 5 * 1024 * 1024},
+	handlers.MediaTypeVideo:       {MaxBytes: 5 * 1024 * 1024},
+	handlers.MediaTypeApplication: {MaxBytes: 5 * 1024 * 1024},
+}
```

Comment:
twilio supports every type we allow users to upload, so just restricting by size here